contractsSol5/KyberStorage.sol

```diff
@@ -227,11 +255,19 @@ contract KyberStorage is IKyberStorage, PermissionGroups2 {
         }
     }
 
-    function getReservesPerTokenSrc(address token) external view returns (bytes32[] memory reserveIds) {
+    function getReservesPerTokenSrc(address token)
```

Comment:
this isn't coding style we use in other files.
I am Ok with this style
just have to be aligned.

which style is it based on?

Reply:
generated by https://github.com/prettier-solidity/prettier-plugin-solidity